res/values/strings_tr.arb

```diff
@@ -0,0 +1,683 @@
+  "cake_pay_subtitle": "İndirimli hediye kartları satın alın (yalnızca ABD)",
+  "cake_pay_web_cards_title": "Cake Pay Web Kartları",
+  "cake_pay_web_cards_subtitle": "Dünya çapında ön ödemeli kartlar ve hediye kartları satın alın",
+  "about_cake_pay": "Cake Pay, Amerika Birleşik Devletleri'ndeki 150.000'den fazla işyerinde anında harcanabilen sanal varlıklarla kolayca hediye kartları satın almanızı sağlar.",
+  "cake_pay_account_note": "Kartları görmek ve satın almak için sadece bir e-posta adresiyle kaydolun. Hatta bazıları indirimli olarak bile mevcut!",
+  "already_have_account": "Zaten bir hesabınız var mı?",
+  "create_account": "Hesap oluştur",
+  "privacy_policy": "Gizlilik Politikası",
+  "welcome_to_cakepay": "Cake Pay'e Hoş Geldiniz!",
+  "sign_up": "Kaydol",
+  "forgot_password": "Parolamı unuttum",
+  "reset_password": "Parolamı sıfırla",
+  "gift_cards": "Hediye kartları",
+  "setup_your_debit_card": "Banka kartını ayarla",
+  "no_id_required": "Kimlik gerekmez. Para yükleyin ve istediğiniz yerde harcayın",
+  "how_to_use_card": "Bu kart nasıl kullanılır",
+  "purchase_gift_card": "Hediye Kartı Satın Al",
+  "verification": "Doğrulama",
+  "fill_code": "Lütfen e-postanıza gelen doğrulama kodunu girin",
+  "dont_get_code": "Kod gelmedi mi?",
+  "resend_code": "Lütfen tekrar gönder",
+  "debit_card": "Ön ödemeli Kart",
+  "cakepay_prepaid_card": "CakePay Ön Ödemeli Kart",
+  "no_id_needed": "Kimlik gerekmez!",
+  "frequently_asked_questions": "Sıkça sorulan sorular",
+  "debit_card_terms": "Ödeme kartı numaranızın (ve kart numaranıza karşılık gelen kimlik bilgilerinin) bu dijital cüzdanda saklanması ve kullanılması, zaman zaman yürürlükte olan ödeme kartı veren kuruluşla yapılan ilgili kart sahibi sözleşmesinin Hüküm ve Koşullarına tabidir.",
+  "please_reference_document": "Daha fazla bilgi için lütfen aşağıdaki belgelere bakınız.",
+  "cardholder_agreement": "Kart Sahibi Sözleşmesi",
+  "e_sign_consent": "E-İmza Onayı",
+  "agree_and_continue": "Kabul Et & Devam Et",
+  "email_address": "E-posta Adresi",
+  "agree_to": "Hesap oluşturarak bunları kabul etmiş olursunuz ",
+  "and": "ve",
+  "enter_code": "Kodu girin",
+  "congratulations": "Tebrikler!",
+  "you_now_have_debit_card": "Artık bir ön ödemeli kartın var",
+  "min_amount" : "Min: ${value}",
+  "max_amount" : "Maks: ${value}",
+  "enter_amount": "Miktar Girin",
+  "billing_address_info": "Eğer fatura adresi istenirse, kargo adresinizi girin",
+  "order_physical_card": "Fiziksel Kart Siparişi",
+  "add_value": "Değer ekle",
+  "activate": "Aktifleştir",
+  "get_a": "Bir ",
+  "digital_and_physical_card": " Dijital para birimleri ile para yükleyebileceğiniz ve ek bilgiye gerek olmayan",
+  "get_card_note": " dijital ve fiziksel ön ödemeli banka kartı edinin!",
+  "signup_for_card_accept_terms": "Kart için kaydol ve koşulları kabul et.",
+  "add_fund_to_card": "Ön ödemeli kartlara para ekle (En fazla yüklenebilir tutar: ${değer})",
```

Comment:
this is causing an error as you translated the parameter ${value} with a word that contains a non-ASCII character 'ğ', the parameters should NOT be translated.

Reply:
also please add the language to the currently supported languages section in the ReadME [here](https://github.com/cake-tech/cake_wallet#current-list-of-language-files)